frontend/src/page/Video.tsx

```diff
@@ -30,8 +36,74 @@ export const VideoPage: React.FC<Props> = ({ id }) => {
 
     const { title, video, description } = event;
     return <>
+        <Player mediaUrl={video} />
         <h1>{title}</h1>
-        <video src={video} controls />
         <p>{description}</p>
     </>;
 };
+
+type PlayerProps = {
+    mediaUrl: string;
+
+    // `true` if this player appears on some content block. `false` if the
+    // player is on the dedicated player page. A `false` just leads to the
+    // player taking up more screen space. Defaults to `false`.
+    embedded?: boolean;
+};
+
+const Player: React.FC<PlayerProps> = ({ mediaUrl, embedded = false }) => {
+    const source = {
+        type: "video" as const,
+        sources: [
+            {
+                src: mediaUrl,
+            },
+        ],
+    };
+
+    const options = {
+        // Compared to the default, "pip" and "airplay" were removed.
+        controls: [
+            // TODO: maybe remove "player-large" -> it's bad for pausing lecture recordings
+            "play-large",
+            "play",
+            "progress",
+            "current-time",
+            "mute",
+            "volume",
+            "captions",
+            "settings",
+            "fullscreen",
+        ],
+        blankVideo: CONFIG.plyr.blankVideo,
+        iconUrl: CONFIG.plyr.svg,
+
+        // TODO:
+        // - `duration`
+        // - `aspectRatio`
+        //
+        // We (will) know these things about the videos, setting them here will
+        // lead to less visual jumps and also earlier duration information.
+    };
+
+    return <>
+        <Global styles={plyrCss as InterpolationWithTheme<any>} />
+        <div css={{
+            // TODO: here we need to adjust the colors of the player based on
+            // the colors from the configuration.
+
+            ...!embedded && {
+                // If this lives on the player page, we want to take the whole
+                // screen width and ignore the padding of the main container.
+                margin: `-${MAIN_PADDING}px -${MAIN_PADDING}px 0 -${MAIN_PADDING}px`,
+
+                // We want to make sure that there is at least a bit of the content
+                // below the video player visible.
+                "& video": { maxHeight: `calc(100vh - ${HEADER_HEIGHT}px - 80px)` },
+                "& .plyr:fullscreen video": { maxHeight: "initial" },
+            },
+        }}>
+            <Plyr source={source} options={options} />
+        </div>
+    </>;
+};
```

Comment:
If we already anticipate this being used in a video content block as well, should this just go in its own module? This module is specifically about the player **page**.